ceph_deploy/mds.py

```diff
@@ -138,6 +138,11 @@ def mds_create(args):
                 distro.release,
                 distro.codename
             )
+            if distro.normalized_name == 'centos':
```

Comment:
I think this is another argument in favor of moving these things into hosts/\*/{rgw,mod,mds}.py, actually.  If it was setup that way, this could be in the ``hosts/rhel`` module, and *that* only applies to RHEL + RHCS.